tests/README.md

```diff
@@ -3,7 +3,7 @@
 
 ## Run the osbuilder tests
 
-osbuilder provides a test script that creates all images and initrds for all
+osbuilder provides a test script that creates all standard and initrd images for all
```

Comment:
any chance we could say `standard (rootfs)` ? if that is the right description...

Reply:
Agreed - updated!

Reply:
Actually everywhere in osbuilder (README, Makefiles,...) rootfs  *tout court* refers to the "rootfs directory", so it would be more correct to call them e.g. disk images.

Reply:
Good point - branch updated yet again! ;)